packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1323,7 +1325,7 @@ curl -v -X GET \
 
 #### List Users with a filter
 
-Lists all users that match the filter criteria
+Lists all users that match the filter criteria.  To ensure optimal performance, we recommend using a [search parameter](#list-users-with-search) instead of a filter.
 
 This operation:
 
```

Comment:
Can we remove "Filters against the most up-to-date data. For example, if you create a user or change an attribute and then issue a filter request, the changes are reflected in your results." ?

Reply:
Yes, we can.